src/devtools/client/webconsole/components/Search/useConsoleSearch.test.tsx

```diff
@@ -89,7 +89,7 @@ describe("useConsoleSearch", () => {
     expect(lastCommitedSearchState?.results.map(result => result.id)).toMatchInlineSnapshot(`
       Array [
         "1",
-        "3",
+        "2",
```

Comment:
:question: How did this snapshot change?

Reply:
The vscode jest extension offered to update this "inline snapshot".
The test and the hook that it tests are a complete mystery to me, far too much React magic. I could only guess from the name that it has to do with searching in the replay console so I did some manual tests and didn't notice any differences in behavior.
@bvaughn could you have a look if this snapshot update makes sense?

Reply:
I'd be happy to look, but this doesn't feel like a very good hand off. Maybe next time we can verify that an unexpected change to snapshot values wasn't a regression before we land the change.

Reply:
Just verified that the committed search result ids still look correct, given the message contents. (The same messages are still being matched, just with slightly different ids).

I think something about this change impacted the ordering of, or id generation for, "newSource" messages in the Redux store. (That's all the ids are.)